website/docs/props/chip.md

```diff
@@ -0,0 +1,173 @@
+## Props
+
+> Also receives all
+> [TouchableNativeFeedback](http://reactnative.dev/docs/touchablenativefeedback.html#props)
+> (Android) or
+> [TouchableOpacity](http://reactnative.dev/docs/touchableopacity.html#props)
+> (iOS) props
+
+- [Props](#props)
+- [Reference](#reference) - [`containerStyle`](#containerstyle) - [`disabled`](#disabled) - [`disabledStyle`](#disabledstyle) - [`disabledTitleStyle`](#disabledtitlestyle) - [`icon`](#icon) - [`iconContainerStyle`](#iconcontainerstyle) - [`iconRight`](#iconright) - [`linearGradientProps`](#lineargradientprops) - [`onPress`](#onpress) - [`raised`](#raised) - [`title`](#title) - [`titleProps`](#titleprops) - [`titleStyle`](#titlestyle) - [`TouchableComponent`](#touchablecomponent) - [`type`](#type) - [`ViewComponent`](#viewcomponent)
```

Comment:
All of these props are already listed in the `Button` component. Don't repeat them, just put a link to the other component. You can see and example of this in other components that inherit their props (like `Searchbar`)

Reply:
Oh okay, I'll update that. I was thinking maybe I'll put it all in one place but yea you're suggestion sounds better!

Reply:
All done! Do let me know if I should leave the prop names in the Reference section as Searchbar does or not.